apis/itemsetapi/itemsetapi.yaml

```diff
@@ -0,0 +1,840 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: Sunbird ItemSet API(s)
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  contact:
+    email: info@sunbird.org
+  description: >-
+      The ItemSet API resources perform operations related to all the _ItemSet_ on the Sunbird Platform. 
+      
+      - Each endpoint serves a different purpose. Depending on your need and purpose, choose the required endpoint
+      
+      - You can use the APIs to perform the Create, Update, Read, Delete, Review and Publish operations. 
+
+      - The URL for Sunbird ItemSet API(s) is `api/itemset/v1`
+host: staging.open-sunbird.org
+basePath: /api/itemset/v1
+securityDefinitions:
+  bearer:
+      type: apiKey  # The arbitrary name for the security scheme
+      in: header # The security scheme parameter is in "header"
+      name: Authorization # The name of the header, query parameter or cookie
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /create:
+    post:
+      description: >-
+       This API is associated with creating an item set on the Sunbird Platform.
+          - The endpoint for **Create ItemSet** is `/create`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+
+      summary: Create ItemSet
+      tags:
+        - ItemSet APIs
+      operationId: Create ItemSet
+      produces:
+        - application/json
+      parameters:
+        - name: Body
+          in: body
+          required: true
+          description: >-
+           'The body parameter refers to the format of the request, the request body (payload) takes the following media types to make the payload:'
+            
+            - JSON 
+            - XML data
+          
+           The body is the representation of the resource object to create an item set (an array of parameters that describes the item set)
+          schema:
+            $ref: '#/definitions/CreateItemSetRequest'
+        - name: Content-Type
+          in: header
+          required: true
+          type: string
+          description:  >-
+            The ItemSet Type entity is the media type of the resource.Possible media types can be:- 
```

Comment:
should this be Content type, not ItemSet Type, as that is what is mentioned against the name parameter?